recipes/ucsc-bed-tools/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set package = "ucsc-bed-tools" %}
```

Comment:
Is it possible to separate the recipe per PR each?

Reply:
What is the reason for doing so, if you don't mind me asking? 

To be honest, I wanted this to be one PR, but the very small file limits are requiring me to split binaries over multiple packages. Do you know if these file size limits can be increased?

Reply:
Because each recipe is usually seen as a separate software/tool, it'd be cleaner to have each as a separate PR; it might also affect CI processes if one of the recipes fails, then the entire PR will fail too. Do you want to include the binary in the recipe or where that has file size limits? I see the test instance of this PR completed successfully for both recipes.

Reply:
I was hoping to make one package that had all of the utilities here:
https://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/
A total of about 3.8GB. 

But the recipe for the failed with out-of-space errors, so I started splitting it into multiple packages, one for recipie for binaries starting bed, another for ones starting with big, etc. 

But ideally, all the binaries at the above URL could be part of one package. Do you know if there is a way to appeal the size limit so I could have one package for all these associated UCSC tools?

Reply:
Ah, I see, probably a core member will have a better idea @bgruening @rpetit3 ?

Reply:
This is currently not possible. Maybe skipping the container generation will help, but then we do not test the package in the container. Which might be fine in that case. But we would need to add support for skipping container builds / testing.